.gitignore

```diff
@@ -22,6 +22,9 @@ node_modules/
 # Eslint cache
 .eslintcache
 
+#CodeCoverage files
```

Comment:
Good find. This exclusion should be like [here](https://github.com/trailheadapps/lwc-recipes/blob/master/.gitignore#L12-L13).

Reply:
fixed!